src/app/GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -307,6 +307,8 @@ public FormCommit(IGitUICommands commands, CommitKind commitKind = CommitKind.No
                 gpgSignCommitToolStripComboBox.SelectedIndex = 0;
             }
 
+            gpgSignCommitToolStripComboBox.ResizeDropDownWidth(minWidth: DpiUtil.Scale(50), maxWidth: DpiUtil.Scale(250));
```

Comment:
in conjunction with #11864
```suggestion
            gpgSignCommitToolStripComboBox.ResizeDropDownWidth(minWidth: 50, maxWidth: 250);
```

Reply:
I was reluctant to change that in advance but ok, fine